code/game/gamemodes/clock_cult/clock_unsorted.dm

```diff
@@ -200,6 +200,22 @@
 #undef GEAR_UNFASTENED
 #undef GEAR_LOOSE
 
+/obj/structure/window/reinforced/clockwork
+	name = "ratvarian window"
+	desc = "A reinforced yellow-glass window, gently luminous."
```

Comment:
I actually intended it as a brass pane thin enough to see through. Glass doesn't really fit ratvar, honestly.